src/collections/_documentation/development/sdk-dev/event-payloads/exception.md

```diff
@@ -15,7 +21,14 @@ except Exception as e:
     raise ValueError() from e
 ```
 
-The order of exceptions would be `Exception` and then `ValueError`.
+The order of exceptions reported would be `Exception` and then `ValueError`.
+
+{% capture __alert_content -%}
+For backwards compatibility, the server also understands events with exceptions
+that are not in the canonical format described here.
```

Comment:
Please do not mention this here. The server accepts a lot of old schemas, but we only document the last one always. We shouldn't encourage SDK authors or customers to send outdated payloads.

Reply:
Understood. Should we move this into an alert on the main page then, since this applies to almost all of the event payload?